nomos/data-availability-layer.md

```diff
@@ -0,0 +1,434 @@
+
+Nodes MAY decide to provide resources to the data availibilty service of the Nomos base-layer or 
+join a Data Availibility Committee within a zone.
+Light nodes are limited resource roles, 
+like a dispersal client or sampling node which utilize Nomos zones to create or retrieve blockchain transactions.
+Compared to other node roles within the network, 
+a light node SHOULD NOT download large amounts of block data owned by zones,
+MAY selectivly vailidate zk-proofs from the Nomos coordination layer,
+MAY verify data availibility of the base layer for zones they prefer.
+
+Data availibity on the base layer is only a temporary guarantee.
+The data can only be verified for a predetermined time, based on the Nomos network.
+The base layer MUST NOT provide long term incentives or 
+allocate resources to repair missing data.
+Zones act as [Data Availability Committees](#) for their own blockchain state.
+Zones SHOULD provide data availability for their blockchain in the event that light nodes can not access data,
+light nodess MAY utilize the data avilability of the Nomos base layer.
+
+### Base-Layer Nodes 
+
+All light node are considered to be Nomos base layer nodes.
+Base-layer nodes MUST NOT process data, 
+but only provide data availability guarantees for a limit amount of time.
+The role of a provider node is to store polynomial commitment schemes for Nomos zones.
+They MUST join a membership based list using libp2p,
+to announce participation in a subnet, a group of provider nodes.
+Nodes must register during a proof-of-validator stage where public keys are verified and 
+a node enters a subnet.
+Nodes registered within a subnet are connected with each other for data passing.
+The list MUST be used by light nodes and 
+zones to connect to a node within a subnet to send data chunk to.
+The data stored by provider nodes MUST not be interpeted or accessed,
+except when sending data for [data availability sampling](#data-sampling), or
+block reconstruction by light clients.
+
+### Message Passing
+
+Nodes that participate in a Nomos zone are considered to be a Nomos base-layer nodes.
+Nomos base-layer utilizes a libp2p publish/subscribe implementation to handle message passing between nodes in the network.
+All base-layer nodes MUST be assigned to a data availability `pubsub-topic`.
+Node configuations SHOULD define a `pubsub-topic` that is shared by all data availiability nodes:
+
+```rs
+pubsub-topic = 'DA_TOPIC';
+```
+
+Communication between different zones with [Data Availability Committees](#) occur directly.
+Zones use a libp2p swarm to connect to different nodes participating in a zone as a validator,
+.
+It is the responsibility of zones to maintain the swarm.
+When a node in the swarm does not provide access to data,
+light nodes MAY use the Nomos data availbilty layer.
+
+#### Sending Data
+
+Zones are responisble for creating data chunks that need to be stored on the blockchain.
```

Comment:
They are responsible for the encoding (reference to the encoding section maybe?).